server/routerlicious/packages/lambdas/src/utils/messageGenerator.ts

```diff
@@ -48,3 +48,30 @@ export const createRoomLeaveMessage = (clientId: string): ISignalMessage => ({
 		content: clientId,
 	}),
 });
+
+export const IRuntimeSignalMessageBody = {
```

Comment:
Can we document this interface, since it's exported?

Reply:
Hm, don't need this one anymore. Killing it. Replaced with simple text string in the `createRuntimeMessage`.`content`.`type`.